src/main/java/edu/cornell/kfs/concur/batch/service/ConcurStandardAccountingExtractService.java

```diff
@@ -9,7 +8,9 @@
 
 public interface ConcurStandardAccountingExtractService {
 
-    ConcurStandardAccountingExtractFile parseStandardAccoutingExtractFileToStandardAccountingExtractFile(File standardAccountingExtractFile) throws ValidationException;
+    ConcurStandardAccountingExtractFile parseStandardAccoutingExtractFileToStandardAccountingExtractFile(String standardAccountingExtractFileName) throws ValidationException;
```

Comment:
This method name could be improved, maybe "ToStandardAccountingExtractFile" can be removed.